static/app/views/metrics/utils/parseMetricWidgetsQueryParam.tsx

```diff
@@ -155,14 +155,15 @@ function parseFormulaWidget(
   widget: Record<string, unknown>,
   baseWidgetParams: BaseWidgetParams
 ): MetricsEquationWidget | null {
-  const formula = parseStringParam(widget, 'formula');
+  const equation =
+    parseStringParam(widget, 'equation') || parseStringParam(widget, 'formula');
```

Comment:
lets just note down somewhere that we should remove this fallback

Reply:
no worries, i noted it down :D

Reply:
Added TODO 😉